.github/workflows/plugin_upstream_testing_base.yml

```diff
@@ -27,6 +27,8 @@ jobs:
         uses: "actions/checkout@v4"
         with:
           ref: "${{ matrix.nautobot-version == 'develop' && 'develop' || matrix.nautobot-version == 'ltm-1.6' && 'ltm-1.6' || env.GITHUB_REF_NAME || 'develop' }}"
+      - name: "Security checks"
+        uses: "./.github/actions/security-checks"
```

Comment:
Is this going to make all Apps start failing the upstream testing until we make a similar change to the App cookiecutter and push it out with driftmanager?

Reply:
Yes - I didn't realize how this workflow was used. I can also remove this.